src/transaction.ts

```diff
@@ -92,6 +91,11 @@ export class Transaction {
      */
     public guardian: string;
 
+    /**
+     * The relayer, in address format, next version all the other addresses will not be string anymore.
+     */
+    public relayer?: Address;
```

Comment:
maybe this doesn't need to be optional. Make it optional on the constructor and if not provided, set to `Address.empty()`?